go.mod

```diff
@@ -94,7 +95,7 @@ require (
 	github.com/multiformats/go-multiaddr v0.4.1
 	github.com/multiformats/go-multiaddr-dns v0.3.1
 	github.com/multiformats/go-multibase v0.0.3
-	github.com/multiformats/go-multicodec v0.3.0
+	github.com/multiformats/go-multicodec v0.3.1-0.20210902112759-1539a079fd61
```

Comment:
We should only be depending on released versions of repos unless there's a documented good reason not to. What's going on here?

Reply:
This was coming from the CAR v2 import. Bumped to v0.4.0.